services/spd/trusty/trusty.c

```diff
@@ -425,6 +445,31 @@ static int32_t trusty_setup(void)
 	if (ret)
 		ERROR("trusty: failed to register fiq handler, ret = %d\n", ret);
 
+	if (aarch32) {
```

Comment:
misra_violation: [Required] MISRA C-2012 Rule 14.4 violation: Condition aarch32 is not essentially boolean.

Change to:
if (aarch32 != 0)

Reply:
changed type